### PR TITLE
[WIP] split triage summarize bq to one day at a time

### PR DESCRIPTION
**Pull request by BenTheElder**
something is still not right here, it gets past choking on the queries now, but it can't parse something. will follow up further later.

log: https://gist.github.com/BenTheElder/f81ef3d704cd4b6d2e6ecf7345e35080

### CODE REVIEW

**Comment by BenTheElder** [comment]
oh and I'm very not proud of this bash

**Comment by BenTheElder** [comment]
we last split the query into two, a simpler update splitting into 3 or 4 parts might work

**Comment by spiffxp** [comment]
ref: https://github.com/kubernetes/test-infra/issues/9271 (I will try repro in a little bit if it helps)

File: triage/update_summaries.sh [hunk 1]
```diff
@@ -42,7 +63,7 @@ curl -sO --retry 6 https://raw.githubusercontent.com/kubernetes/kubernetes/maste
 
 mkdir -p slices
 
-pypy summarize.py triage_builds.json triage_tests_{1,2}.json \
+pypy summarize.py triage_builds.json triage_tests_{1,2,3,4,5,6,7}.json \
```
[the remark this answers is not in the document]
Comment (BenTheElder) [reply triage/update_summaries.sh]: ok, current bug is parsing the contents of one of this, will look into finding out which and why first

**Comment by rmmh** [comment]
This is getting pretty gross :(

Why not output to a temporary table, export it to GCS, and then download it? 

    bq query --allow_large_results --headless -n0 --replace --destination_table k8s-gubernator:temp.triage "select path build, test.name name, test.failure_text failure_text from [k8s-gubernator:build.week] where test.failed and timestamp_to_sec(started) > TIMESTAMP_TO_SEC(DATE_ADD(CURRENT_DATE(), -7, 'DAY'))"
    bq extract --compression GZIP --destination_format NEWLINE_DELIMITED_JSON 'k8s-gubernator:temp.triage' gs://k8s-gubernator/triage-latest.json.gz
    gsutil cp gs://k8s-gubernator/triage-latest.json.gz triage-latest.json.gz
    gzip -d triage-latest.json.gz
    wc -l triage-latest.json
    > 113852

**Comment by BenTheElder** [comment]
SGTM, I was just poking at getting the job to not hang as oncall, I'm not very familiar with BQ. I'll circle back to this ~tonight or tomorrow and try that instead.

**Comment by spiffxp** [comment]
/close
I think this got addressed under #9271

**Pull request closed without merging**